addons/account/views/account_analytic_view.xml

```diff
@@ -17,8 +17,8 @@
                     <group name="amount" position="after">
                         <group/> <!-- put Accounting group under Amount group -->
                         <group name="accounting" string="Accounting">
-                            <field name="general_account_id"/>
-                            <field name="move_id"/>
+                            <field name="general_account_id" options="{'no_create': True}"/>
```

Comment:
Is there also an issue when trying to create an account?

Reply:
No, this was a change that was incorporated in 13 and 14, so I supposed the person who made the change didn't do it for 12. Probably a fix that was made in v13 and ported forward, but not for v12. Maybe not a good idea to put it in the same commit, but I figured they were tackling the same issue where you create an account that needs more than just a name.